examples/simple_eval.py

```diff
@@ -0,0 +1,69 @@
+from parsy import digit, generate, match_item, regex, string, success, test_item
+
+
+def lexer(code):
+    whitespace = regex(r'\s*')
+    integer = digit.at_least(1).map(''.join).map(int)
+    float_ = (
+        digit.many() + string('.').result(['.']) + digit.many()
+    ).map(''.join).map(float)
+    parser = whitespace >> ((
+        integer | float_ | regex(r'[()*/+-]')
+    ) << whitespace).many()
+    return parser.parse(code)
+
+
+def eval_tokens(tokens):
+    # parse and evaluate at the same time.
```

Comment:
This _looks_ too much like a comment to the following two lines. Since it's not, I'd at least separate it with an additional blank line.